src/runtime/virtcontainers/network.go

```diff
@@ -220,6 +221,40 @@ func NewNetwork(configs ...*NetworkConfig) (*Network, error) {
 	}, nil
 }
 
+func LoadNetwork(netInfo persistapi.NetworkInfo) *Network {
```

Comment:
Not for changing in this PR per se, but I think it'd be better if we had our own defined type for NetworkInfo in this package, and had persistapi use this (instead of the opposite and relying on that otherwise unrelated package). Would you have any concerns/input on this, if I were to change this in a follow on PR?